src/api/DataApi.test.js

```diff
@@ -35,6 +38,18 @@ const MOCK_ENTITY_DATA = {
   [genRandomUUID()]: ['value_3', 'value_4'],
 };
 
+const MOCK_BINARY_OBJECT_REQUEST = {
+  value: {
```

Comment:
Same comment as the one above. It doesn't make a difference at the moment, but this object is malformed.